python312Packages.authlib: 1.3.1 -> 1.3.2

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for python312Packages.authlib is: Library for building OAuth and OpenID Connect servers

meta.homepage for python312Packages.authlib is: https://github.com/lepture/authlib

meta.changelog for python312Packages.authlib is: https://github.com/lepture/authlib/blob/v1.3.2/docs/changelog.rst


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 1.3.2 with grep in /nix/store/z4izhkci22nhfw705gsmch4w5rwmcnbx-python3.12-authlib-1.3.2
- found 1.3.2 in filename of file in /nix/store/z4izhkci22nhfw705gsmch4w5rwmcnbx-python3.12-authlib-1.3.2

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
45 total rebuild path(s)

45 package rebuild(s)

First fifty rebuilds by attrpath

chatgpt-retrieval-plugin
fit-trackee
home-assistant-component-tests.discovergy
home-assistant-component-tests.lamarzocco
home-assistant-component-tests.point
home-assistant-component-tests.senz
home-assistant-component-tests.vicare
khoj
matrix-synapse
matrix-synapse-plugins.matrix-synapse-ldap3
matrix-synapse-plugins.matrix-synapse-mjolnir-antispam
matrix-synapse-plugins.matrix-synapse-s3-storage-provider
matrix-synapse-plugins.matrix-synapse-shared-secret-auth
matrix-synapse-unwrapped
mealie
open-webui
pgadmin4
pgadmin4-desktopmode
poetryPlugins.poetry-audit-plugin
powerdns-admin
python311Packages.aiosenz
python311Packages.authlib
python311Packages.flask-security
python311Packages.gradio
python311Packages.kanidm
python311Packages.lmcloud
python311Packages.private-gpt
python311Packages.pydiscovergy
python311Packages.pypoint
python311Packages.pyvicare
python311Packages.safety
python311Packages.weaviate-client
python312Packages.aiosenz
python312Packages.authlib
python312Packages.flask-security
python312Packages.gradio
python312Packages.kanidm
python312Packages.lmcloud
python312Packages.pydiscovergy
python312Packages.pypoint
python312Packages.pyvicare
python312Packages.safety
python312Packages.weaviate-client
safety-cli
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/z4izhkci22nhfw705gsmch4w5rwmcnbx-python3.12-authlib-1.3.2 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A python312Packages.authlib https://github.com/r-ryantm/nixpkgs/archive/892313fc242ee83dbed86dadfc2329024ef460b3.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/892313fc242ee83dbed86dadfc2329024ef460b3#python312Packages.authlib
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/z4izhkci22nhfw705gsmch4w5rwmcnbx-python3.12-authlib-1.3.2
ls -la /nix/store/z4izhkci22nhfw705gsmch4w5rwmcnbx-python3.12-authlib-1.3.2/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages marked as broken and skipped:</summary>
  <ul>
    <li>chatgpt-retrieval-plugin</li>
    <li>chatgpt-retrieval-plugin.dist</li>
    <li>khoj</li>
    <li>khoj.dist</li>
  </ul>
</details>
<details>
  <summary>5 packages failed to build:</summary>
  <ul>
    <li>fit-trackee</li>
    <li>fit-trackee.dist</li>
    <li>powerdns-admin</li>
    <li>python311Packages.private-gpt</li>
    <li>python311Packages.private-gpt.dist</li>
  </ul>
</details>
<details>
  <summary>70 packages built:</summary>
  <ul>
    <li>home-assistant-component-tests.discovergy</li>
    <li>home-assistant-component-tests.lamarzocco</li>
    <li>home-assistant-component-tests.point</li>
    <li>home-assistant-component-tests.senz</li>
    <li>home-assistant-component-tests.vicare</li>
    <li>matrix-synapse</li>
    <li>matrix-synapse-plugins.matrix-synapse-ldap3</li>
    <li>matrix-synapse-plugins.matrix-synapse-ldap3.dist</li>
    <li>matrix-synapse-plugins.matrix-synapse-mjolnir-antispam</li>
    <li>matrix-synapse-plugins.matrix-synapse-mjolnir-antispam.dist</li>
    <li>matrix-synapse-plugins.matrix-synapse-s3-storage-provider</li>
    <li>matrix-synapse-plugins.matrix-synapse-s3-storage-provider.dist</li>
    <li>matrix-synapse-plugins.matrix-synapse-shared-secret-auth</li>
    <li>matrix-synapse-plugins.matrix-synapse-shared-secret-auth.dist</li>
    <li>matrix-synapse-unwrapped</li>
    <li>matrix-synapse-unwrapped.dist</li>
    <li>mealie</li>
    <li>mealie.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>pgadmin4</li>
    <li>pgadmin4-desktopmode</li>
    <li>pgadmin4-desktopmode.dist</li>
    <li>pgadmin4.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>python311Packages.aiosenz</li>
    <li>python311Packages.aiosenz.dist</li>
    <li>python311Packages.authlib</li>
    <li>python311Packages.authlib.dist</li>
    <li>python311Packages.flask-security</li>
    <li>python311Packages.flask-security.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.kanidm</li>
    <li>python311Packages.kanidm.dist</li>
    <li>python311Packages.lmcloud</li>
    <li>python311Packages.lmcloud.dist</li>
    <li>python311Packages.pydiscovergy</li>
    <li>python311Packages.pydiscovergy.dist</li>
    <li>python311Packages.pypoint</li>
    <li>python311Packages.pypoint.dist</li>
    <li>python311Packages.pyvicare</li>
    <li>python311Packages.pyvicare.dist</li>
    <li>python311Packages.safety</li>
    <li>python311Packages.safety.dist</li>
    <li>python311Packages.weaviate-client</li>
    <li>python311Packages.weaviate-client.dist</li>
    <li>python312Packages.aiosenz</li>
    <li>python312Packages.aiosenz.dist</li>
    <li>python312Packages.authlib</li>
    <li>python312Packages.authlib.dist</li>
    <li>python312Packages.flask-security</li>
    <li>python312Packages.flask-security.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.kanidm</li>
    <li>python312Packages.kanidm.dist</li>
    <li>python312Packages.lmcloud</li>
    <li>python312Packages.lmcloud.dist</li>
    <li>python312Packages.pydiscovergy</li>
    <li>python312Packages.pydiscovergy.dist</li>
    <li>python312Packages.pypoint</li>
    <li>python312Packages.pypoint.dist</li>
    <li>python312Packages.pyvicare</li>
    <li>python312Packages.pyvicare.dist</li>
    <li>safety-cli (python312Packages.safety)</li>
    <li>safety-cli.dist (python312Packages.safety.dist)</li>
    <li>python312Packages.weaviate-client</li>
    <li>python312Packages.weaviate-client.dist</li>
  </ul>
</details>

---

###### Maintainer pings

cc @flokli for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

dotlambda commented on pkgs/development/python-modules/authlib/default.nix:
```diff
@@ -18,7 +18,7 @@
   pname = "authlib";
-  version = "1.3.1";
+  version = "1.3.2";
   format = "setuptools";
```
```suggestion
  pyproject = true;
```

dotlambda commented on pkgs/development/python-modules/authlib/default.nix:
```diff
@@ -27,7 +27,7 @@ buildPythonPackage rec {
+    hash = "sha256-gaFnai5QzHhnyn73JB+QzybaolLWC9barBFdnlEMyMU=";
   };
 
   propagatedBuildInputs = [
```
```suggestion
  dependencies = [
```

dotlambda reviewed (commented):
`requests` should be removed

dotlambda committed:
python312Packages.authlib: 1.3.1 -> 1.3.2

Diff: https://github.com/lepture/authlib/compare/refs/tags/v1.3.1...v1.3.2

Changelog: https://github.com/lepture/authlib/blob/v1.3.2/docs/changelog.rst

dotlambda committed:
fit-trackee: unpin authlib

dotlambda committed:
python312Packages.flask-security: tests depend on requests

dotlambda committed:
python312Packages.weaviate-client: update dependencies

dotlambda commented:
Result of `nixpkgs-review pr 338531` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages marked as broken and skipped:</summary>
  <ul>
    <li>chatgpt-retrieval-plugin</li>
    <li>chatgpt-retrieval-plugin.dist</li>
    <li>khoj</li>
    <li>khoj.dist</li>
  </ul>
</details>
<details>
  <summary>8 packages failed to build:</summary>
  <ul>
    <li>home-assistant-component-tests.vicare</li>
    <li>powerdns-admin</li>
    <li>python311Packages.private-gpt</li>
    <li>python311Packages.private-gpt.dist</li>
    <li>python311Packages.pyvicare</li>
    <li>python311Packages.pyvicare.dist</li>
    <li>python312Packages.pyvicare</li>
    <li>python312Packages.pyvicare.dist</li>
  </ul>
</details>
<details>
  <summary>67 packages built:</summary>
  <ul>
    <li>fit-trackee</li>
    <li>fit-trackee.dist</li>
    <li>home-assistant-component-tests.discovergy</li>
    <li>home-assistant-component-tests.lamarzocco</li>
    <li>home-assistant-component-tests.point</li>
    <li>home-assistant-component-tests.senz</li>
    <li>matrix-synapse</li>
    <li>matrix-synapse-plugins.matrix-synapse-ldap3</li>
    <li>matrix-synapse-plugins.matrix-synapse-ldap3.dist</li>
    <li>matrix-synapse-plugins.matrix-synapse-mjolnir-antispam</li>
    <li>matrix-synapse-plugins.matrix-synapse-mjolnir-antispam.dist</li>
    <li>matrix-synapse-plugins.matrix-synapse-s3-storage-provider</li>
    <li>matrix-synapse-plugins.matrix-synapse-s3-storage-provider.dist</li>
    <li>matrix-synapse-plugins.matrix-synapse-shared-secret-auth</li>
    <li>matrix-synapse-plugins.matrix-synapse-shared-secret-auth.dist</li>
    <li>matrix-synapse-unwrapped</li>
    <li>matrix-synapse-unwrapped.dist</li>
    <li>mealie</li>
    <li>mealie.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>pgadmin4</li>
    <li>pgadmin4-desktopmode</li>
    <li>pgadmin4-desktopmode.dist</li>
    <li>pgadmin4.dist</li>
    <li>poetryPlugins.poetry-audit-plugin</li>
    <li>poetryPlugins.poetry-audit-plugin.dist</li>
    <li>python311Packages.aiosenz</li>
    <li>python311Packages.aiosenz.dist</li>
    <li>python311Packages.authlib</li>
    <li>python311Packages.authlib.dist</li>
    <li>python311Packages.flask-security</li>
    <li>python311Packages.flask-security.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.kanidm</li>
    <li>python311Packages.kanidm.dist</li>
    <li>python311Packages.lmcloud</li>
    <li>python311Packages.lmcloud.dist</li>
    <li>python311Packages.pydiscovergy</li>
    <li>python311Packages.pydiscovergy.dist</li>
    <li>python311Packages.pypoint</li>
    <li>python311Packages.pypoint.dist</li>
    <li>python311Packages.safety</li>
    <li>python311Packages.safety.dist</li>
    <li>python311Packages.weaviate-client</li>
    <li>python311Packages.weaviate-client.dist</li>
    <li>python312Packages.aiosenz</li>
    <li>python312Packages.aiosenz.dist</li>
    <li>python312Packages.authlib</li>
    <li>python312Packages.authlib.dist</li>
    <li>python312Packages.flask-security</li>
    <li>python312Packages.flask-security.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.kanidm</li>
    <li>python312Packages.kanidm.dist</li>
    <li>python312Packages.lmcloud</li>
    <li>python312Packages.lmcloud.dist</li>
    <li>python312Packages.pydiscovergy</li>
    <li>python312Packages.pydiscovergy.dist</li>
    <li>python312Packages.pypoint</li>
    <li>python312Packages.pypoint.dist</li>
    <li>safety-cli (python312Packages.safety)</li>
    <li>safety-cli.dist (python312Packages.safety.dist)</li>
    <li>python312Packages.weaviate-client</li>
    <li>python312Packages.weaviate-client.dist</li>
  </ul>
</details>

dotlambda commented:
> 8 packages failed to build

Pyvicare will be replaced with pyvicare-neo in #338722. The other failures aren't new.
